scripts/ci/40_e2e.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 set -euxo pipefail
 
-yarn run e2e $1 --headless --production --record --group=$1
+HOST="http://localhost:4200"
+
+ yarn start-server-and-test "yarn start $(echo ${1%-*})" $HOST "yarn run e2e $1 --headless --production --record --group=$1 --base-url $HOST --devServerTarget '' "
```

Comment:
Pull the `echo ${1%-*}` to a variable and explain why you can't simply use `$1`.

Reply:
Maybe also pull the `yarn start ...` and `yarn run ....` strings to variables as well for readability.

Reply:
And add a newline at the end of the file :)

Reply:
Would be nice to see an example of how you run this script.

Another thing that I'm wondering...
is the api mocked?
if not, you might have to run multiple instances before running the tests. If `start-server-and-test` doesn't support that, then we already have a script that waits for ports to become active: `scripts/_wait-for.sh`

Reply:
The apiis mocked in all cypress tests as the environment variables `API_MOCKS` and `NX_API_MOCKS` are set in the github workflow. Wondering if they should be set in this file as well...? Probably?

Reply:
No longer applicable changes.